content/docs/start/data-and-model-versioning.md

```diff
@@ -247,6 +247,16 @@ defines data file versions. Git itself provides the version control. DVC in turn
 creates these `.dvc` files, updates them, and synchronizes DVC-tracked data in
 the <abbr>workspace</abbr> efficiently to match them.
 
+## Model versioning
+
+Apart from data files, DVC eases the way you work with models. Models in a
+project usually change more frequently than data files and they need to be kept
+in sync with changes in other elements of a project. Model files are no
+different than data files when it comes to tracking their versions. DVC also
+provides means to track minor changes in model files without fully checking in
+to underlying VCS. In later sections of this series, you'll see how DVC enables
+to track changes in pipelines consisting of multiple model and data files.
```

Comment:
Interesting. What's the purpose of this section?

Maybe we can just make sure we mention models enough in the existing text e.g. early on near "To start tracking a file or directory" we could say ", whether its raw data or a trained ML model"

I'm not opposed to also adding the section but it's unclear to me as of now what added value it has i.e. what is special about versioning model files vs. any large file?

Reply:
My bet that @iesahin was trying to make it extra explicit. My 2cs - I'm fine to keep it, be 100% that we have all parts covered now :)

Reply:
I'm not opposed to keeping it but the text is a bit repetitive, I'd like to read what makes versioning model files special. I don't have a more specific suggestion for now...

Reply:
Actually the gist is _models are also files and tracked as files_ and normally I wouldn't consider mentioning this explicitly. However it looks we need to be more explicit here. I think _model_ seems a rather abstract concept in user's mind and _you can keep track of your models_ as a separate section seems a better way for handling for #2096.

Reply:
> the gist is models are also files and tracked as files and normally

Perfect, let's focus on that 🙂. Preferably as the opening sentence: "Model files are no different than data files when it comes to tracking their versions."

**Then** add the model-specific info like "models change more frequently than data".

Reply:
Other comments:

> provides means to track minor changes

What are we referring to here?

> In later sections of this series, you'll see how DVC enables to track changes in pipelines

The comment makes sense in general but probably not for this **Model versioning** section. Let's just focus on the task and skip it, I think.

Thanks

Reply:
> What are we referring to here?

I'm thinking about run-cache, parameters etc. Features that work separately from Git. These are related to models rather than data.

Reply:
I updated and removed `pipelines` in b46cca31 .

Reply:
Hey @iesahin sorry, this one fell through the cracks during the 2.0 release... Checking now. May take over if you're no longer available for this.